Bynder/Sdk/Query/Asset/AssetUsageQuery.cs

```diff
@@ -0,0 +1,58 @@
+using Bynder.Sdk.Query.Decoder;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Bynder.Sdk.Query.Asset
+{
+    /// <summary>
+    /// Operation on the Asset Usage API.
+    /// </summary>
+    public class AssetUsageQuery
+    {
+        /// <summary>
+        /// Initializes the class with asset usage information.
+        /// </summary>
+        /// <param name="integrationId">ID of the integration that the usage applies to.</param>
+        /// <param name="assetId">ID of the asset that the usage applies to.</param>
+        /// <param name="uri">Location of the asset usage.</param>
+        /// <param name="additional">Additional information.</param>
+        public AssetUsageQuery(string integrationId, string assetId, string uri = null, string additional = null)
+        {
+            IntegrationId = integrationId;
+            AssetId = assetId;
+            Uri = uri;
+            Additional = additional;
+        }
```

Comment:
For consistency with the other queries, I would leave the optional fields out of the constructor:
```suggestion
        /// <summary>
        /// Initializes the class with asset usage information.
        /// </summary>
        /// <param name="integrationId">ID of the integration that the usage applies to.</param>
        /// <param name="assetId">ID of the asset that the usage applies to.</param>
        public AssetUsageQuery(string integrationId, string assetId)
        {
            IntegrationId = integrationId;
            AssetId = assetId;
        }
```